packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Style the Okta-hosted Sign-In Widget
+---
+
+You can add any HTML, CSS, or JavaScript to the **Customization** page that you want. You can also customize the sign-in page per application and with multiple brands.
```

Comment:
I was confused by what the Customization page was, since it hadn't been introduced. Would it work to say instead: "You can add any HTML, CSS, or JavaScript you want to the sign-in page"?

Reply:
I think it would be good for this intro paragraph to also mention a couple of other topics covered in this page, like macros. Just to give a sense of what's coming up.